dih_models/parameters.py

```
@@ -9203,24 +9213,38 @@ def _pd_compound_fv_abolition(C, g, r, N):
-STATE_RTT_TREATMENT_ACCELERATION_SUFFERING_HOURS = Parameter(
+STATE_RTT_TREATMENT_ACCELERATION_SUFFERING_YEARS = Parameter(
     float(STATE_RTT_TREATMENT_ACCELERATION_DALYS)
-    * float(GLOBAL_YLD_PROPORTION_OF_DALYS)
-    * HOURS_PER_YEAR,
+    * float(GLOBAL_YLD_PROPORTION_OF_DALYS),
+    manual_ref="knowledge/appendix/state-right-to-trial-impact.qmd",
+    source_type="calculated",
+    description="Conditional disability-equivalent years prevented by the treatment schedule shift. This is the years-lived-with-disability share of averted DALYs, not a claim that every year represents maximum conscious pain.",
+    display_name="Disability-Equivalent Suffering Years Prevented by Universal Right to Try with Evidence",
+    unit="years",
+    formula="STATE_RTT_TREATMENT_ACCELERATION_DALYS × GLOBAL_YLD_PROPORTION_OF_DALYS",
```

Ruff (0.16.2)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)